### ISSUE TITLE
To access updated State in parent component

### ISSUE THREAD

**Issue by Himanshu-uidev** [express-labs/pure-react-carousel#348]
The props passed to CarouselProvider renders the initial carousel, but there should be a way to access the updated state in parent component, I faced the issue while implementing accessibility on the carousel, while tabbing if we have passed slides the navigation buttons for example **ButtonBack, ButtonNext** are not changing there state as they do when we click on them.
There must be an updated state available in parent component.

**Comment by captnstarburst** [comment]
Hey @Himanshu-uidev ! I hope I understood you correctly, you should be able to see the carousels current state by utilizing the [Higher Order Component withStore](https://www.npmjs.com/package/pure-react-carousel#withstore-higher-order-component) or [Context CarouselContext](https://www.npmjs.com/package/pure-react-carousel#withstore-higher-order-component) and I created an example using withStore [here](https://codesandbox.io/s/friendly-breeze-d30m2?file=/src/MyComponent.js). You also said you ran into a bug while tabbing ? Would you mind providing an example for that?
